python/paddle/v2/framework/tests/test_sequence_slice_op.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import numpy as np
+import sys
+from op_test import OpTest
+
+class TestSequenceSliceOp(OpTest):
+    def set_data(self):
+        # only supprot one level LoD
+        x = np.random.random((100, 3, 2)).astype('float32')
+        lod = [[0, 20, 40, 60, 80, 100]]
+        offset = np.array([1, 2, 3, 4, 5]).flatten().astype("int64")
+        length = np.array([10, 8, 6, 4, 2]).flatten().astype("int64")
```

Comment:
You can separate the definitions of data. Like [this](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/framework/tests/test_conv2d_op.py#L80)